src/forms/redux-form/TimeInput.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import TimeInput from '../TimeInput';
+
+/**
+ * This component wraps the standard TimeInput for use with redux-form.
+ * It deconstructs props that redux-form sets and re-sets them on TimeInput.
+ * @param {object} props React component props
+ * @return {React.Component} TimeInput
+ */
+const ReduxFormTimeInput = props => {
+	const { input, meta, ...other } = props;
+
+	return <TimeInput {...input} error={meta.error} {...other} />;
+};
+
+ReduxFormTimeInput.displayName = 'ReduxFormTimeInput';
```

Comment:
for debugging in the console I believe. 
the file name of redux form components are just `redux-form/TimeInput.jsx`
mike spencer gave me that feedback before that we should have a displayName.